kms/attestations/noxfile.py

```diff
@@ -0,0 +1,312 @@
+# Copyright 2019 Google LLC
```

Comment:
Please remove this file, add `noxfile_config.py` file instead.

Reply:
Done